docs/book/src/topics/node-outbound-lb.md

```diff
@@ -0,0 +1,64 @@
+# Node Outbound Load Balancer
+
+This document describes how to configure your clusters' node outbound load balancer.
+
+### Public Clusters
+
+For public clusters ie. clusters with api server load balancer type set to `Public`, CAPZ automatically configures a node outbound load balancer with the default settings.
+
+To provider custom settings for the node outbound load balacer, use the `nodeOutboundLB` section in cluster configuration.
+
+Here is an example of a node outbound load balancer with `frontendIPsCount` set to 3. CAPZ will read this value and create 3 front end ips for this load balancer.
```

Comment:
Let's include a node about why a user would want to increase the number of frontent IPs, something like:

> You may want more than one outbound IP address if you are running a large cluster that is processing lots of connections. See [here](https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-outbound-connections#multifesnat) for more documentation about how adding more outbound IP addresses can increase the number of SNAT ports available for use by the Standard Load Balancer in your cluster.